.github/workflows/build.yml

```diff
@@ -100,6 +100,12 @@ jobs:
       - name: Run Archetype Tests
         run: |
           ./mvnw -B clean install -Parchetype-test -pl :foundation-archetype
+      - name: Publish aggregated Cucumber report
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs
+          destination_dir: cucumber-html-reports
```

Comment:
A: I think this will end up in the URL, so I'd prefer something like `test-reports`.

Reply:
Updated the `destination_dir`.